businesscentral-monitoring/modules/businesscentral-monitoring/module.yaml

```diff
@@ -25,7 +25,7 @@ ports:
 - value: 8001
 artifacts:
 - name: "rhpam_business_central_monitoring_distribution.zip"
```

Comment:
@spolti I suppose the name/md5 will be changed in a different PR?
if so, I suppose `rhpam_business_central_monitoring_distribution.zip` should be renamed to `bamoe_business_central_monitoring_distribution.zip` (also the other zips like add-ons.zip, business-central-eap7-deployable.zip).

In case you need, bamoe CR1 artifacts are already available in candidates folder.

Reply:
I'll update it after we finish this first renaming step.